install_config/aggregate_logging.adoc

```diff
@@ -416,24 +416,84 @@ The deployer creates an ephemeral deployment in which all of a pod's data is
 lost upon restart. For production usage, add a persistent storage volume to each
 Elasticsearch deployment configuration.
 
-The following example specifies a volume for an Elasticsearch replica (using a
-xref:../architecture/additional_concepts/storage.adoc#persistent-volume-claims[PersistentVolumeClaim]):
+The best-performing volumes are local disks, if it is possible to use
+them. There are some complications with doing so.
 
+. The relevant service account must be given the privilege to mount and edit a local volume, as follows:
++
 ====
 ----
-$ oc volume dc/logging-es-rca2m9u8 \
+$ oadm policy add-scc-to-user privileged  \
```

Comment:
@pweil- I tried `hostmount-anyuid` first and it did not have access due to SELinux context. I believe it's much the same problem we had with fluentd - https://github.com/openshift/origin-aggregated-logging/issues/89#issuecomment-204127737

It seems like less-than-privileged may be possible, but I'm not quite sure how and it seems like it would be a PITA for a user to set up. What do you think?